### PR TITLE
Add USBCDC_ECM example

### PR DESCRIPTION
**Pull request by juhaylinen**

### CODE REVIEW

File: USB/USBCDC_ECM/README.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+## USBCDC_ECM example
```
Comment (AnotherButler) [inline USB/USBCDC_ECM/README.md]: One the API documentation merges, I'll update this README to point to it in a new PR.

**Comment by AnotherButler** [comment]
ping @0xc0170

File: USB/USBCDC_ECM/main.cpp [hunk 2]
```diff
@@ -0,0 +1,27 @@
+#include "mbed.h"
```
Comment (0xc0170) [inline USB/USBCDC_ECM/main.cpp]: we dont include license headers to the code files? Like this one, we have this documented in our license guide. If anyone copies this, should also copy the copyright + license. PLease add

Comment (0xc0170) [reply USB/USBCDC_ECM/main.cpp]: I've checked some files, some have some don't . All should so please add it here and should be LGTM

**Comment by AnotherButler** [comment]
@0xc0170 Can you please merge this? It's telling me I can't.

**Comment by 0xc0170** [comment]
>@0xc0170 Can you please merge this? It's telling me I can't.

You should be able to, if not, please email us (maintainers) , will review

**Comment by AnotherButler** [comment]
Thank you 👍  If it doesn't let me again, I'll let you know.

**Pull request merged**